### DISCUSSION TITLE
Easyinstall script no longer working for the OLED Control Board since November update?

### DISCUSSION THREAD

**Discussion by InfraredRidingHood** [akuker/RASCSI#1445] [General]
Hi, I was updating my PiSCSI Zero with the OLED Control Board when I realized the Easyinstall would only work for the Web interface and would no longer activate the Control Board.  Is there another method of installing the Control Board interface or is another update coming shortly?

**Comment by rdmark** [comment]
We're preparing to tag a new stable release that resolves this issue. There are just some last-minute bugs that we need to look closer at.

The breakage to the Control Board client was fixed with https://github.com/PiSCSI/piscsi/pull/1394
